test/docusign/constructor/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os/signal"
+	"syscall"
+
+	"github.com/amp-labs/connectors/common"
+	connTest "github.com/amp-labs/connectors/test/docusign"
+	"github.com/amp-labs/connectors/test/utils"
+)
+
+func main() {
+	// Handle Ctrl-C gracefully.
+	ctx, done := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	defer done()
+
+	// Set up slog logging.
+	utils.SetupLogging()
+
+	// Post authentication information can be cached and reused for every connector instance.
+	info := getPostAuthInfo(ctx)
+
+	conn := connTest.GetDocusignConnector(ctx)
+	defer utils.Close(conn)
+
+	err := conn.SetPostAuthInfo(ctx, info)
+	if err != nil {
+		utils.Fail("error setting post authentication metadata", "error", err)
+	}
+
+	fmt.Println("constructor finished")
+}
+
+func getPostAuthInfo(ctx context.Context) *common.PostAuthInfo {
+	conn := connTest.GetDocusignConnector(ctx)
+	defer utils.Close(conn)
+
+	info, err := conn.GetPostAuthInfo(ctx)
```

Comment:
I couldn't manage to get a working `creds.json` to test `GetPostAuthInfo` method. I am having issues with Docusign account.